integration/kubernetes/helm-chart/alluxio/templates/worker/daemonset.yaml

```diff
@@ -84,8 +87,6 @@ spec:
             valueFrom:
               fieldRef:
                 fieldPath: status.hostIP
-          - name: ALLUXIO_WORKER_JAVA_OPTS
-            value: {{ print " -Dalluxio.worker.hostname=$(ALLUXIO_WORKER_HOSTNAME) " | quote }}
```

Comment:
@cheyang I don't think you should remove this `ALLUXIO_WORKER_JAVA_OPTS`. This handles `alluxio.worker.hostname` property, which can only be decided at runtime. Please restore these 2 lines.

Reply:
`ALLUXIO_WORKER_JAVA_OPTS ` in configmap can also handle this  property. The ALLUXIO_WORKER_HOSTNAME can be determined in runtime.